pwndbg/checksec.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+ Wrapper to checksec utility.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+import subprocess
+
+import pwndbg.file
+
+def checksec():
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    result = {}
+
+    for program in ['checksec', 'checksec.sh']:
```

Comment:
@disconnect3d ,my code is taken directly from the module commands/checksec.py. I don't hate particularly the `if program`, but if this is you preferred coding style let's modify it in both position. ( pwndbg/checksec.py and commands/checksec.py )